lib/appliance_console/external_httpd_configuration.rb

```diff
@@ -70,7 +71,7 @@ def unconfigure_httpd
         config_file_write(config, HTTPD_CONFIG, timestamp)
 
         say("Restarting httpd ...")
-        LinuxAdmin::Service.new("httpd").restart
+        LinuxAdmin::Service.new(ENV['MIQ_APACHE_SERVICE_NAME']).restart
```

Comment:
Instead of referencing the env directly, can we move into a method or maybe set once as a constant early on?  Additionally, what if the ENV var is not set...do we at least default to httpd?